MyPlayground copy.playground/Contents.swift

```diff
@@ -0,0 +1,105 @@
+import UIKit
+
+var Mycity = "Medinah"
```

Comment:
Try to use swift naming standards for variables. Here its recommended to have like 
**var myCity = "Madinah"**

i.e. start with small case then make first letter of each word Upper case.

Reply:
1. Adopt camelCase pattern. 
2. `var Mycity = "Dubi"` < is missing!